Vester/Tests/DSCluster/Rename-datastore.Vester.ps1

```diff
@@ -0,0 +1,32 @@
+# Test file for the Vester module - https://github.com/WahlNetwork/Vester
+# Called via Invoke-Pester VesterTemplate.Tests.ps1
+
+# Test title, e.g. 'DNS Servers'
+$Title = 'Rename default datastore'
+
+# Test description: How New-VesterConfig explains this value to the user
+$Description = 'default datastore starts with datastore* name which has to be changed to hostname specific datastore'

```

Comment:
The datastore name doesn't "have to" be named differently, it's just opentable policy to do so. This should be worded to reflect that, such as "the default datastore name _can_ be changed"